index.html

```diff
@@ -403,6 +403,64 @@ <h3>
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          Steps for when a user changes payment method
+        </h3>
+        <p class="note" data-lt="BasicCardRequest">
+          When the end user explicitly switches from one card to another, these
+          steps cause an event to fire in the <cite data-cite=
+          "!payment-request">Payment Request API</cite>. The algorithm
+          presupposes that the user agent is only presenting <a>supported
+          cards</a> to the end user, by having filtered out unsupported cards
+          based on the initiating payment request's <a>BasicCardRequest</a>'s
+          <a>supportedNetworks</a> and <a>supportedTypes</a> values.
+        </p>
+        <p>
+          The <dfn>steps for when a user changes payment method</dfn> are as
+          follows. To mitigate fingerprinting concerns, the user agent MUST NOT
+          run these steps unless the a user explicitly switches to a different
+          card by performing some user action (e.g., by selecting a different
+          card explicitly from a list of cards). For cards that are preselected
+          by default by the user agent, any matching <code><a data-cite=
+          "payment-request#dom-paymentdetailsmodifier">PaymentDetailsModifier</a></code>
+          apply instead (without the need to run these steps).
+        </p>
+        <ol>
+          <li>From the <a>supported cards</a> presented to the end user, let
+          <var>card</var> be the <a>card</a> the user selected.
+          </li>
+          <li>Let <var>methodDetails</var> be a newly created
+          <a>BasicCardChangeDetails</a> dictionary.
+          </li>
+          <li>Set <var>methodDetails</var>["<a>type</a>"] to the
+          <a>BasicCardType</a> that represents <var>card</var>'s <a>type</a>.
+          </li>
+          <li>Run the payment request's <a data-cite=
+          "!payment-request#dfn-payment-method-changed-algorithm">payment
+          method changed algorithm</a> with <var>methodDetails</var> and
+          "basic-card".
```

Comment:
The algorithm as defined in https://github.com/w3c/payment-request/pull/695/files does not take a second argument, but it probably should.

Reply:
Yeah, I was deriving the "method name" in a hand-wavy way in the Payment Request spec. I agree this explicit way is better.

Reply:
Updated https://github.com/w3c/payment-request/pull/695 to take the second argument.